ironfish/src/rpc/routes/chain/getNetworkHashPower.ts

```diff
@@ -7,63 +7,67 @@ import { ValidationError } from '../../adapters'
 import { ApiNamespace, router } from '../router'
 
 export type GetNetworkHashPowerRequest = {
-  lookup?: number // number of blocks to lookup
-  height?: number // estimate network speed at the time of the given height
+  blocks?: number // number of blocks to look back
+  sequence?: number // the sequence of the latest block from when to estimate the network speed
 }
 
 export type GetNetworkHashPowerResponse = {
   hashesPerSecond: number
+  blocks: number // The actual number of blocks used in the hash rate calculation
+  sequence: number // The actual sequence of the latest block used in hash rate calculation
 }
 
 export const GetNetworkHashPowerRequestSchema: yup.ObjectSchema<GetNetworkHashPowerRequest> =
   yup
     .object({
-      lookup: yup.number().optional(),
-      height: yup.number().optional(),
+      blocks: yup.number().optional(),
+      sequence: yup.number().optional(),
     })
     .defined()
 
 export const GetNetworkHashPowerResponseSchema: yup.ObjectSchema<GetNetworkHashPowerResponse> =
   yup
     .object({
       hashesPerSecond: yup.number().defined(),
+      blocks: yup.number().defined(),
+      sequence: yup.number().defined(),
     })
     .defined()
 
 router.register<typeof GetNetworkHashPowerRequestSchema, GetNetworkHashPowerResponse>(
   `${ApiNamespace.chain}/getNetworkHashPower`,
   GetNetworkHashPowerRequestSchema,
   async (request, node): Promise<void> => {
-    let lookup = request.data?.lookup ?? 120
-    const height = request.data?.height ?? -1
+    let blocks = request.data?.blocks ?? 120
+    const sequence = request.data?.sequence ?? -1
 
     /*
-      For bitcoin, a negative lookup specifies using all blocks since the last difficulty change.
+      For bitcoin, a negative blocks specifies using all blocks since the last difficulty change.
       For ironfish, the difficulty changes for every block, so this isn't supported.
     */
```

Comment:
I don't think we need this in our docs since it's documenting something we don't support. 

Also in our chain, negative sequences are generally supported and just relative to the head.

Reply:
Added support for negative sequences